docs/use_cases/met_tool_wrapper/PlotDataPlane/PlotDataPlane_grib1.py

```diff
@@ -0,0 +1,114 @@
+"""
+PlotDataPlane: GRIB1 Input
+==========================
+
+met_tool_wrapper/PlotDataPlane/PlotDataPlane_grib1.conf
+
+"""
+##############################################################################
+# Scientific Objective
+# --------------------
+#
+# Generate a postscript image to test if the input data can be read by the MET
+# tools.
+
+##############################################################################
+# Datasets
+# --------
+#
+# | **Input:** Sample GRIB1 file
+#
+# | **Location:** All of the input data required for this use case can be found in the met_test sample data tarball. Click here to the METplus releases page and download sample data for the appropriate release: https://github.com/dtcenter/METplus/releases
+# | This tarball should be unpacked into the directory that you will set the value of INPUT_BASE. See 'Running METplus' section for more information.
+#
+# | **Data Source:** Unknown
```

Comment:
It looks like from the name of the output file, the data is coming from the NAM. Either this line can be updated, or line 19 can have the additional information of NAM and this line can be deleted.
*This comment can be applied to all of the use case config files*

Reply:
I added NAM to the data source line.